### ISSUE TITLE
Segmentation fault when train semantic3d

### ISSUE THREAD

**Issue by 08822407d** [QingyongHu/RandLA-Net#115]
I ran the model in tensorflow-gpu docker container(I have the latest version of docker, and the latest version of Nvidia-docker, the docker environment is CUDA9.0 and tensorflow-gpu, and my machine is notebook with i7-9750H and RTX-3000 and 32GB RAM), when training semantic3d, Segmentation fault (core dumped) happend, and the tty output was:

> Preparing reprojection indices for validation and test
finished
Initiating input pipelines
WARNING:tensorflow:From /home/cheyh/AImodels/RandLA-Net/RandLANet.py:265: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

>Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

>See `tf.nn.softmax_cross_entropy_with_logits_v2`.

>/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/gradients_impl.py:108: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
****EPOCH 0****
Segmentation fault (core dumped)

I only change batch_size to 2 since my GPU only has 6GB memory, i didn't change other cinfigs,
before crash, GPU memory usage was about 4GB, and machine RAM usage was up to 21GB,
does anyone have idea about the problem?

**Comment by changhexingchen** [comment]
@08822407d Have you solved the problem which is Segmentation fault (core dumped)? I encountered the same problem.

**Comment by 08822407d** [comment]
It seems to be the problem of laptop version rtx3000, I changed to desktop GTX-1080Ti, it works whitout such error now
